modules/EsminiAdapter/src/esminiadapter.cpp

```diff
@@ -123,18 +127,11 @@ void EsminiAdapter::onConnectedObjectIdsMessage(const ConnectedObjectIds::messag
 
 //! Message queue callbacks
 
-void EsminiAdapter::onAbortMessage(const Abort::message_type::SharedPtr) {
+void EsminiAdapter::onStaticAbortMessage(const Abort::message_type::SharedPtr) {
```

Comment:
Shouldn't the name of this method be "closeScenarioEngine"? Right now it is named based on what will call it, not what it does.